packages/bundler-plugin-core/src/types.ts

```diff
@@ -139,6 +139,23 @@ export interface Options {
    * Defaults to `false`
    */
   dryRun?: boolean;
+
+  /** Options for OIDC authentication. */
+  oidc?: {
+    /**
+     * Whether to use GitHub OIDC for authentication.
+     *
+     * Defaults to `false`
+     */
+    useGitHubOIDC: boolean;
+
+    /**
+     * The OIDC endpoint to use for authentication.
+     *
+     * Defaults to `https://codecov.io`
+     */
+    OIDCEndpoint?: string;
```

Comment:
(See other PR comment re: this variable.)

Technically, something like below may be a little clearer:

    /**
     * The expected audience in the GitHub OIDC JWT token to be validated by the apiUrl.
     *
     * Defaults to `https://codecov.io`
     */
    gitHubOIDCTokenAudience?: string;
The value of this string needs to match that at codecov-api [here](https://github.com/codecov/codecov-api/blob/615e4f385133b3c2f6a1c04109d1ef77cbe948c8/upload/helpers.py#L251).

We are never actually hitting this "endpoint" and technically the value of this doesn't need to be a url at all.